front/lib/api/assistant/generation.ts

```diff
@@ -83,6 +84,33 @@ export async function constructPromptMultiActions(
 
   // INSTRUCTIONS section
   let instructions = "INSTRUCTIONS:\n";
+
+  if (
```

Comment:
> I thought we wanted this for the global agent only?

Yes please it would be much preferable to have this on the global agent. The change would be [here](https://github.com/dust-tt/dust/blob/2e3c53bb1a5c16fd6d895332bdc183b617a59f6a/front/lib/api/assistant/global_agents.ts#L560)